src/main/interactive_shell.h

```diff
@@ -25,9 +25,12 @@
 namespace CVC4 {
 
 class Command;
-class ExprManager;
 class Options;
 
+namespace api {
+class Solver;
```

Comment:
Since we are repeating this pattern in multiple places, maybe we should have a `cvc4cppfwd.h` with all the forward declarations (similar to [iosfwd](https://en.cppreference.com/w/cpp/header/iosfwd))?

Reply:
It's only one single forward declaration, not sure if it make sense for that?